### PR TITLE
jfrog-cli: add head build

### PR DESCRIPTION
**Pull request by timsutton**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/j/jfrog-cli.rb [hunk 1]
```diff
@@ -25,8 +26,10 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jf -v")
-    assert_match version.to_s, shell_output("#{bin}/jfrog -v")
+    if build.stable?
+      assert_match version.to_s, shell_output("#{bin}/jf -v")
+      assert_match version.to_s, shell_output("#{bin}/jfrog -v")
+    end
```
Comment (chenrui333) [inline Formula/j/jfrog-cli.rb:32]: I dont think we need to do this, there is head build test.

Comment (chenrui333) [reply Formula/j/jfrog-cli.rb:32]: we dont test head build per CI so no need to add it.

Comment (chenrui333) [reply Formula/j/jfrog-cli.rb:32]: also head build is a moving target, there is no need to test it either.

Comment (chenrui333) [reply Formula/j/jfrog-cli.rb:32]: just drop the test block change, which should be good.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**